src/main/java/org/broadinstitute/hellbender/tools/funcotator/filtrationRules/AlleleFrequencyGnomadUtils.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.tools.funcotator.filtrationRules;
+
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Stream;
+
+class AlleleFrequencyGnomadUtils {
+    /**
+     * Sub-population suffixes used within gnomAD. Used for calculating max MAF.
+     */
+    private enum GnomadSubpopSuffixes {
+        afr, // African / African-American
+        afr_female,
+        afr_male,
+        amr, // Latino
+        amr_female,
+        amr_male,
+        asj, // Ashkenazi Jewish
+        asj_female,
+        asj_male,
+        eas, // East Asian
+        eas_female,
+        eas_jpn,
+        eas_kor,
+        eas_male,
+        eas_oea,
+        female,
+        fin, // Finnish
+        fin_female,
+        fin_male,
+        male,
+        nfe, // Non-Finnish European
+        nfe_bgr,
+        nfe_est,
+        nfe_female,
+        nfe_male,
+        nfe_nwe,
+        nfe_onf,
+        nfe_seu,
+        nfe_swe,
+        oth, // Other
+        oth_female,
+        oth_male,
+        popmax,
+        raw,
+        sas, // South Asian
+        sas_female,
+        sas_male
+    }
+
+    /**
+     * Prefix for gnomAD genome allele-frequency Funcotations.
+     */
+
+    private enum GnomadDataset {
+        gnomAD_genome, gnomAD_exome;
+
+        // Prefix for gnomAD allele-frequency Funcotations.
+        private String alleleFrequencyPrefix = name() + "_AF_";
+
+        // Name of annotations for whether gnomAD records for a particular data type were filtered out.
+        private String filterAnnotation = name() + "_FILTER";
+    }
+
+    /**
+     * Calculate the max MAF across all gnomAD sub-populations from the given Funcotations.
+     * <p>
```

Comment:
Orphan paragraph tag?

Reply:
Editor put it in. Javadoc <p> tags don't have to have a closing tag. Its dumb.

Reply:
Weird.  OK, cool.